packages/compass-aggregations/src/components/stage-preview/stage-preview.jsx

```diff
@@ -79,11 +79,13 @@ class StagePreview extends Component {
             <div className={styles['stage-preview-out-text']}>
               Documents persisted to collection specified by $merge.
             </div>
-            <div
+            <Link
```

Comment:
Some accessibility issues caught by eslint that were easy enough to change right away